websocket/polygon.go

```diff
@@ -154,41 +154,23 @@ func (c *Client) Unsubscribe(topic Topic, tickers ...string) error {
 	return nil
 }
 
+// Output returns the next message in the output queue. If no messages are available, it returns nil.
 func (c *Client) Output() any {
-	return <-c.output
```

Comment:
this was a blocking call which could cause users to deadlock if the client isn't receiving data